benchmarks/inference/results/output.txt

```diff
@@ -0,0 +1,102 @@
+=============================================================================
```

Comment:
Do you actually want to checkin this file?

Reply:
I'll remove it and add the results back when I make the runner output things in tabular form(?)